tests/Fixtures/TestBundle/Entity/FooDummy.php

```diff
@@ -35,11 +37,21 @@ class FooDummy
     #[ORM\Id]
     #[ORM\GeneratedValue(strategy: 'AUTO')]
     private ?int $id = null;
+
     /**
      * @var string The foo name
      */
     #[ORM\Column]
     private $name;
+
+    #[ApiProperty(writable: false)]
+    #[ORM\Column(nullable: true)]
+    public $nonWritableProp;
+
+    #[ApiProperty(readableLink: true, writableLink: true)]
```

Comment:
Unfortunatelly they are not `true` by default. If not setting these attributes, there will be a skolem IRI in the JSON instead.

It would be a great behaviour for Embeddables to treat them more like properties than relations. Could you assist in working on that? But I would recommend to tackle this in a separate issue and PR.

Reply:
embeddables were hard to handle previously because if they're marked as a resource the behavior was different. Now I think it'd be possible to do this indeed.